web/modules/custom/dpl_example_content/content/eventseries/f6aafa27-a538-44a3-ae1f-02f8efbfb068.yml

```diff
@@ -0,0 +1,252 @@
+_meta:
+  version: '1.0'
+  entity_type: eventseries
+  uuid: f6aafa27-a538-44a3-ae1f-02f8efbfb068
+  bundle: default
+  default_langcode: en
+  depends:
+    93887b31-0c4b-495f-9131-ef8b87aee1c9: eventinstance
+    695728ab-b1ff-4131-afed-ac1f01cfdb88: eventinstance
+    92e444d2-9333-4a04-ac8a-ec851e1aa3d8: eventinstance
+    ff338d06-676a-4755-a54b-c9cf93568d25: eventinstance
+    a94a3e0d-032a-4e3c-a0fa-415b4adb0d39: eventinstance
+    5c70dd69-6b91-47c3-92d5-7b48478595f8: eventinstance
+    5cfedbfc-9daa-4f23-8c9a-86fce0ecd7d9: eventinstance
+    0d92b355-81e6-4260-bb68-84046111b1ad: eventinstance
+    d7ee955b-3eb1-4680-a2ea-479308054948: eventinstance
+    ae781b38-f2e1-418e-949d-59c32663d4b8: eventinstance
+    2195956f-7aaf-46fa-83e1-554d8ec5be5c: eventinstance
+    6dabaa46-4d6d-488a-9f4d-06f7cd09789a: eventinstance
+    0fa7245a-3434-4d96-9793-a1848a22d37a: taxonomy_term
+    ca329ad9-5a1c-4f55-b8a9-a60843b66466: media
+    e4ce7616-976d-43fc-aed2-a01c42b67db9: media
+    e8283d3c-968a-4e74-ae02-9f02b76bb839: media
+    f0204f0b-1f63-4c76-bb67-e3f7489ee392: media
+default:
+  revision_uid:
+    -
+      target_id: 1
+  status:
+    -
+      value: true
+  uid:
+    -
+      target_id: 1
+  title:
+    -
+      value: 'Fars legestue'
+  recur_type:
+    -
+      value: monthly_recurring_date
+  consecutive_recurring_date:
+    -
+      value: ''
+      end_value: ''
+      time: '08:00 am'
+      end_time: '11:30 pm'
+      duration: 5
+      duration_units: minute
+      buffer: 0
+      buffer_units: minute
+  daily_recurring_date:
+    -
+      value: ''
+      end_value: ''
+      time: '08:00 am'
+      duration: 900
+      end_time: '08:00 am'
+      duration_or_end_time: duration
+  weekly_recurring_date:
+    -
+      value: '2024-01-01T13:49:30'
+      end_value: '2030-01-01T13:49:30'
+      time: '08:00 am'
+      duration: 900
+      end_time: '08:00 am'
+      duration_or_end_time: duration
+      days: monday
+  monthly_recurring_date:
+    -
+      value: '2024-01-01T13:49:30'
+      end_value: '2025-01-01T13:49:30'
```

Comment:
Please reconsider this values

It is nice to have a recurring event to generate some instances to populate `/events`. Setting the year 2024 is not helpful. We risk over time that the dates pass and thus no longer shown on the list.

 I suggest setting a value in the far future where the recurring event will interleave with the existing single events.